### DISCUSSION TITLE
Add CLI arguments for window size and position

### DISCUSSION THREAD

**Discussion by hbashton** [Chatterino/chatterino2#3651] [Ideas]
I've recently been using the [streamlink-twitch-gui](https://streamlink.github.io/streamlink-twitch-gui/) application to watch Twitch, and while **VLC allows me to provide a window size and position via CLI, Chatterino does not**. This extra step of resizing the Chatterino window is unnecessary. I propose an update which adds command line arguments for window size and position, in the form of  --width=int & --height=int, --window-x=int & --window-y=int respectively. These would fall under the VLC standard, which is the default [Streamlink](https://streamlink.github.io/streamlink-twitch-gui/) application. Please consider this feature, and anyone who reads this please consider giving it a bump as it is very much needed to make [Streamlink](https://streamlink.github.io/streamlink-twitch-gui/) a seamless desktop integrator with Twitch.

**Comment by Felanbird** [comment]
Agreed - also would be useful for issue reports when a `-c` instance is started, which I do sometimes.
